### DISCUSSION TITLE
Make field searching more usable for non-admins

### DISCUSSION THREAD

**Discussion by timkelty** [craftcms/cms#8456] [Ideas]
Field searching (e.g. body:"salty dog"), can be useful in the control panel, but it requires the user to know the field handle, which is never exposed to non-admins.

Most frequently, this request comes up with a relation field.
User will be viewing an entry with a related element, and want to view elements with that same relation.

They ask me, and the best I can do is tell them to search for `relatedFieldHandle:"The Element Title"`.

Related: https://github.com/craftcms/cms/issues/1084

**Comment by brandonkelly** [comment]
This will probably get better with #1630
